scripts/deploy/verifyContract.ts

```diff
@@ -1,32 +0,0 @@
-import hr from 'hardhat'
```

Comment:
Do not remove this because it is a generic script to validate different contracts

Reply:
Added it back, does nothing now as the only thing verified seems to be the tiers contract.